src/pages/search/index.tsx

```diff
@@ -139,13 +103,66 @@ function getIsOpenNow(times: Time[]): boolean {
   return isOpenNow;
 }
 
+type OpeningHoursInfoBlockProps = {
+  openingHours: UnifiedSearchOpeningHours;
+  openAt: Date | undefined;
+  locale: Locale;
+};
+
+function OpeningHoursInfoBlock({
+  openingHours,
+  openAt,
+  locale,
+}: OpeningHoursInfoBlockProps): React.ReactElement<
+  React.ComponentProps<typeof InfoBlock>
+> | null {
+  const { t } = useTranslation("search_page");
+
+  const openingHourTimes =
+    (openAt
+      ? openingHours?.data?.find(
+          (timeData) => timeData.date === format(openAt, "yyyy-MM-dd")
+        )?.times
```

Comment:
These brackets confuse me. Aren't they redundant?

Reply:
If I understand correctly, it was unclear where the bracket before `openAt` ends. The bracket exists so that we can use a default value for the entire Elvis expression:

```
(openAt
      ? oneValue
      : twoValue) ?? []
```

Instead of using a single variable, I used two to remove the complex brackets.

Reply:
Yeah you are correct here, brackets were needed :)